core/src/epicli/data/common/ansible/playbooks/roles/applications/tasks/wait-for-cluster.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
As I saw we use such task also in `kubernetes` roles, in `upgrade`, `utils` dir we also have something similar. Is it a time to remove duplicates? If we decide so, I can also make changes in my PR.

Reply:
Good point. Let me think about it :+1:

Reply:
After some extra heavy refactor I have only 2 task files now: one for checking if cluster is available and second one for deploying an app. I think it's better to keep those 2 files and not introduce more roles imported from elsewhere. I vote for applications feature to be self-contained. :)